init/wp_ipc_advice.sql

```diff
@@ -1,37 +1,37 @@
+  KEY `Section51Enquiry` (`Section51Enquiry`)
+) DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
+
+
+INSERT INTO ipclive.wp_ipc_advice (`AdviceID`,`FolderName`,`EnquiryDate`,`EnquiryMethod`,`IndustrySector`,`CaseReference`,`EnqFirstName`,`EnqLastName`,`EnquiryOrganisation`,`EnquiryDetail`,`AdviceGiven`,`RespondedBy`,`Section51Enquiry`,`InitiatedDate`,`DateEnquiryReceived`,`DateAdviceGiven`,`Attachments`,`DateCreated`,`DateLastModified`) VALUES ('WA033004-Advice-00001',NULL,NULL,'Email',NULL,'EN010120','Vijaya','Vasantha','Test General Advice','This is a general advice test republish again and again to test the general advie under register of advices and not under the case as the case is not yet published to the NI Website.','This is a general advice test republish again and again to test the general advie under register of advices and not under the case as the case is not yet published to the NI Website.',NULL,'Yes',NULL,NULL,'2020-03-20',NULL,'2020-03-20 08:00:26','2022-08-05 08:24:31');
+INSERT INTO ipclive.wp_ipc_advice (`AdviceID`,`FolderName`,`EnquiryDate`,`EnquiryMethod`,`IndustrySector`,`CaseReference`,`EnqFirstName`,`EnqLastName`,`EnquiryOrganisation`,`EnquiryDetail`,`AdviceGiven`,`RespondedBy`,`Section51Enquiry`,`InitiatedDate`,`DateEnquiryReceived`,`DateAdviceGiven`,`Attachments`,`DateCreated`,`DateLastModified`) VALUES ('EN043005-Advice-00006',NULL,NULL,'Email',NULL,'EN010120','Vijaya','Vasantha','Test Organisation','This is a enquiry to republish again and again','This is the Depublish',NULL,'Yes',NULL,NULL,'2020-03-20','EN043005-000023','2020-03-23 15:43:39','2022-08-05 08:24:31');
+INSERT INTO ipclive.wp_ipc_advice (`AdviceID`,`FolderName`,`EnquiryDate`,`EnquiryMethod`,`IndustrySector`,`CaseReference`,`EnqFirstName`,`EnqLastName`,`EnquiryOrganisation`,`EnquiryDetail`,`AdviceGiven`,`RespondedBy`,`Section51Enquiry`,`InitiatedDate`,`DateEnquiryReceived`,`DateAdviceGiven`,`Attachments`,`DateCreated`,`DateLastModified`) VALUES ('WA023013-Advice-00001',NULL,NULL,'Meeting',NULL,'EN010120','Vijaya','Vasantha','Test Organisation','This is my advice to check johns query','Just create and view whether it appears on IP Exports document',NULL,'Yes',NULL,NULL,'2020-04-03',NULL,'2020-04-03 10:02:12','2022-08-05 08:24:31');
+INSERT INTO ipclive.wp_ipc_advice (`AdviceID`,`FolderName`,`EnquiryDate`,`EnquiryMethod`,`IndustrySector`,`CaseReference`,`EnqFirstName`,`EnqLastName`,`EnquiryOrganisation`,`EnquiryDetail`,`AdviceGiven`,`RespondedBy`,`Section51Enquiry`,`InitiatedDate`,`DateEnquiryReceived`,`DateAdviceGiven`,`Attachments`,`DateCreated`,`DateLastModified`) VALUES ('WA023013-Advice-00005',NULL,NULL,'Email',NULL,'TR040009','Vijaya Krishna','Vasantha','Test Organisation','Advice for Testing Version History','This is modfiying the existing to check the status posted as Modified',NULL,'Yes',NULL,NULL,'2020-04-16','TR040009-Advice-00001-1, TR040009-Advice-00002-1, TR040009-Advice-00003-1','2020-04-16 05:28:39','2022-08-05 08:24:31');
```

Comment:
I think the attachment Id's are likely to be linked to the Advice Id, so for WA023013-Advice-00005, we would have
'WA023013-Advice-00005-1, WA023013-Advice-00005-2, WA023013-Advice-00005-3'

Reply:
Ok, I have created some docs to match